app/src/main/java/com/danielkim/soundrecorder/fragments/FileViewerFragment.java

```diff
@@ -44,10 +45,10 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View v = inflater.inflate(R.layout.fragment_file_viewer, container, false);
 
-        RecyclerView mRecyclerView = (RecyclerView) v.findViewById(R.id.recyclerView);
+        RecyclerView mRecyclerView = v.findViewById(R.id.recyclerView);
         mRecyclerView.setHasFixedSize(true);
         LinearLayoutManager llm = new LinearLayoutManager(getActivity());
-        llm.setOrientation(LinearLayoutManager.VERTICAL);
+        llm.setOrientation(RecyclerView.VERTICAL);
```

Comment:
why change the constant here? is `LinearLayoutManager` class compatible with constants of `RecyclerView` class?